next-jdbc/src/ragtime/next_jdbc.clj

```diff
@@ -41,8 +61,10 @@
                        table_name)))
 
 (defn- table-exists? [datasource ^String table-name]
-  (some (partial metadata-matches-table? table-name)
-    (get-table-metadata datasource)))
+  (let [{:keys [tables quote]} (get-db-metadata datasource)
+        table-name-unquoted (unquote-table-name quote table-name)]
+    (some (partial metadata-matches-table? table-name-unquoted)
+      tables)))
```

Comment:
Could you indent this as:

```clojure
    (some (partial metadata-matches-table? table-name-unquoted)
          tables)
```

The original indent is wrong, and since these lines are being changed, it might as well be fixed here.